spec/calendar.html

```diff
@@ -1366,12 +1366,20 @@ <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1
         1. Set _additionalFields_ to ? ToObject(_additionalFields_).
         1. Let _additionalFieldsCopy_ be OrdinaryObjectCreate(*null*).
         1. Perform ? CopyDataProperties(_additionalFieldsCopy_, _additionalFields_, « », « *undefined* »).
-        1. NOTE: Every property of _additionalFieldsCopy_ is a data property with non-*undefined* value, but some property keys may be Symbols.
+        1. NOTE: Every property of _fieldsCopy_ and _additionalFieldsCopy_ is an enumerable data property with non-*undefined* value, but some property keys may be Symbols.
         1. Let _additionalKeys_ be ! _additionalFieldsCopy_.[[OwnPropertyKeys]]().
         1. Assert: _calendar_.[[Identifier]] is *"iso8601"*.
-        1. Let _ignoredKeys_ be ISOFieldKeysToIgnore(_additionalKeys_).
+        1. Let _overriddenKeys_ be ISOFieldKeysToIgnore(_additionalKeys_).
         1. Let _merged_ be OrdinaryObjectCreate(*null*).
-        1. Perform ! CopyDataProperties(_merged_, _fieldsCopy_, _ignoredKeys_, « *undefined* »).
+        1. NOTE: The following steps ensure that property iteration order of _merged_ matches that of _fields_ as modified by omitting overridden properties and appending non-overlapping properties from _additionalFields_ in iteration order.
```

Comment:
is there a reason not to upstream this improvement into CopyDataProperties itself?

Reply:
This only comes up when multiple objects are being merged, whereas [CopyDataProperties](https://tc39.es/ecma262/multipage/abstract-operations.html#sec-copydataproperties) is concerned with a _single_ source. Are you aware of any other need for such a generalization?

Reply:
ah, i suppose not - Object.assign merges multiple objects but I'm not sure if the semantics match.

Either way tho, you're using this in two places in this PR - perhaps it should still be extracted to an AO?

Reply:
It's not actually two places, sup-temporal.calendar.prototype.mergefields overrides sec-temporal.calendar.prototype.mergefields and all such overrides will be going away after the current PRs settle and we non-ISO-8601 calendar operations move from ECMA-402 to ECMA-262.